Makefile

```diff
@@ -98,12 +98,10 @@ test-performance-locust-querystring-search-ci:
 
 .PHONY: Code Analysis
 code-analysis:  ## Code Analysis
```

Comment:
@wesleybl Is there any reason to keep this command in the Makefile? We can probably remove it entirely.

Reply:
I think whoever put the make command was not to use it in the CI. I even think that make was never used in CI. I think it was for local execution, without changing the files. But I will remove it then.